ext/web/timers.rs

```diff
@@ -85,9 +85,18 @@ pub async fn op_sleep(
   millis: u64,
   rid: ResourceId,
 ) -> Result<bool, AnyError> {
+  // If a timer is requested with <50ms resolution, request the high-res timer.
```

Comment:
Maybe explain in the comment why 50ms and not 15ms

Reply:
I bumped it up to 100ms and added some text around it. It's a bit of a wild guess at what's appropriate, but 100ms seems like a good cutoff for "caring more about timer precision".